.github/workflows/pr-checklist.yaml

```diff
@@ -0,0 +1,30 @@
+name: PR-checklist
```

Comment:
Maybe add a comment before like
#alt: we used to have a .github/pull_request_template.md file, but then it causes X and Y
# so we opted for a workflow that does not have problem X and Y.

I usually like to put more context on why we decided to do something, and why the obvious naive alternative is not great.

Reply:
True. Done.